PSReadLine/KeyBindings.cs

```diff
@@ -328,6 +330,8 @@ void SetDefaultEmacsBindings()
                 { Keys.AltPeriod,       MakeKeyHandler(YankLastArg,          "YankLastArg") },
                 { Keys.AltUnderbar,     MakeKeyHandler(YankLastArg,          "YankLastArg") },
                 { Keys.CtrlAltY,        MakeKeyHandler(YankNthArg,           "YankNthArg") },
+                { Keys.AltH,            MakeKeyHandler(ShowParameterHelp,      "ShowParameterHelp") },
+                { Keys.F1,              MakeKeyHandler(ShowCommandHelp,        "ShowCommandHelp") }
```

Comment:
```suggestion
                { Keys.AltH,            MakeKeyHandler(ShowParameterHelp,    "ShowParameterHelp") },
                { Keys.F1,              MakeKeyHandler(ShowCommandHelp,      "ShowCommandHelp") }
```
Please align the third column

Reply:
Fixed